snap/snapcraft.yaml

```diff
@@ -356,9 +376,21 @@ parts:
       - -usr/share/mime
       - -usr/share/pkgconfig
       - -usr/share/thumbnailers
-    organize:
-      usr/bin/xfce4-terminal.wrapper: bin/x-terminal-emulator
+    override-build: |
+      craftctl default
+      export PATH="/root/parts/desktop-gui/build/flutter-distro/bin:$HOME/.pub-cache/bin:$PATH"
+      rm -rf /root/parts/desktop-gui/build/flutter-distro
+      git clone --depth 1 --branch 3.16.9 https://github.com/flutter/flutter.git /root/parts/desktop-gui/build/flutter-distro
+      flutter config --suppress-analytics --no-enable-web --enable-linux-desktop --no-enable-macos-desktop --no-enable-windows-desktop --no-enable-android --no-enable-ios --no-enable-fuchsia --no-enable-custom-devices
+      flutter precache --linux
 
-  glue:
-    plugin: dump
-    source: snap-wrappers
+      dart pub global activate protoc_plugin
+      export MP_PROTO_FILE_DIR=$HOME/parts/multipass/src/src/rpc
+
+      mkdir -p lib/generated
+      /root/parts/multipass/build/vcpkg_installed/x64-linux-release/tools/protobuf/protoc --dart_out=grpc:lib/generated -I $MP_PROTO_FILE_DIR -I /root/parts/multipass/build/_deps/grpc-src/third_party/protobuf/src $MP_PROTO_FILE_DIR/multipass.proto google/protobuf/timestamp.proto
+
+      flutter pub get
+      flutter build linux --release --verbose --target lib/main.dart
+    organize:
+      /root/parts/desktop-gui/build/build/linux/x64/release/bundle: bin/
```

Comment:
I'm not a real fan of **all** of the parts of the GUI bundle being dumped into `bin/`.  It's ok for now, but we should do better in another PR.